feature-libs/checkout/root/config/default-checkout-routing-config.ts

```diff
@@ -5,11 +5,13 @@ export const defaultCheckoutRoutingConfig: RoutingConfig = {
     routes: {
       checkoutLogin: { paths: ['checkout-login'], authFlow: true },
       checkout: { paths: ['checkout'] },
+      checkoutPaymentType: { paths: ['checkout/payment-type'] },
```

Comment:
these were removed, but wasn't supposed to be since it's the old checkout

Reply:
Hmm, not sure what are you saying here? 
I compared it with develop, and the files are identical. Perhaps you addressed it in the meantime, and thank you for that :)